### ISSUE TITLE
Github Actions stopped working

### ISSUE THREAD

**Issue by omrilotan** [gorangajic/isbot#49]
Workflow output:
```
- Workflows can't be executed on this repository. Please check your payment method or billing status.
```

Workflows are limited to 2,000 minutes/month. [Recent issue](https://github.community/t5/GitHub-Actions/GitHub-Actions-workflows-can-t-be-executed-on-this-repository/td-p/38153) suggests this may be limiting open repositories.

I can move CI process to CircleCI which support pipeline as code

**Comment by omrilotan** [comment]
@gorangajic Any thoughts? I've seen a lot of issues around Github starting Nov 13

**Comment by omrilotan** [comment]
Resolved in #52
